data/database.py

```diff
@@ -1901,6 +1901,53 @@ class RepoMirrorConfig(BaseModel):
     root_rule = ForeignKeyField(RepoMirrorRule)
 
 
+@unique
+class IndexStatus(IntEnum):
+    """
+    Possible statuses of manifest security scan progress.
+    """
+
+    FAILED = -1
+    UNSUPPORTED = 0
+    TIMED_OUT = 1
+    WAITING = 2
```

Comment:
If we're going to stick with the smaller terms, can we put comments explaining what they mean? What does `WAITING` mean here? Waiting to start? To complete?